2022-03-31 07:31:50,350 INFO    Thread-43 :29951 [wandb_setup.py:_flush():75] Loading settings from /home/ltorres/.config/wandb/settings
2022-03-31 07:31:50,350 INFO    Thread-43 :29951 [wandb_setup.py:_flush():75] Loading settings from wandb/settings
2022-03-31 07:31:50,350 INFO    Thread-43 :29951 [wandb_setup.py:_flush():75] Loading settings from environment variables: {'project': 'cloud classification', 'entity': 'leonardtd', 'root_dir': '/home/ltorres/leo/tesis/cloud-classification/code', 'run_id': '925yo3hw', 'sweep_param_path': '/home/ltorres/leo/tesis/cloud-classification/code/wandb/sweep-03otgj81/config-925yo3hw.yaml', 'sweep_id': '03otgj81'}
2022-03-31 07:31:50,350 INFO    Thread-43 :29951 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-03-31 07:31:50,350 INFO    Thread-43 :29951 [wandb_init.py:_log_setup():405] Logging user logs to /home/ltorres/leo/tesis/cloud-classification/code/wandb/run-20220331_073150-925yo3hw/logs/debug.log
2022-03-31 07:31:50,350 INFO    Thread-43 :29951 [wandb_init.py:_log_setup():406] Logging internal logs to /home/ltorres/leo/tesis/cloud-classification/code/wandb/run-20220331_073150-925yo3hw/logs/debug-internal.log
2022-03-31 07:31:50,350 INFO    Thread-43 :29951 [wandb_init.py:_jupyter_setup():355] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f685d451820>
2022-03-31 07:31:50,351 INFO    Thread-43 :29951 [wandb_init.py:init():439] calling init triggers
2022-03-31 07:31:50,351 INFO    Thread-43 :29951 [wandb_init.py:init():442] wandb.init called with sweep_config: {'batch_size': 26, 'criterion': 'loge', 'device': 'cuda:2', 'epochs': 25, 'hidden_dim': 512, 'learning_rate': 4.594822028254274e-05, 'num_heads': 1, 'num_hidden': 2, 'optimizer': 'adam', 'similarity_threshold': 0.6}
config: {}
2022-03-31 07:31:50,351 INFO    Thread-43 :29951 [wandb_init.py:init():492] starting backend
2022-03-31 07:31:50,351 INFO    Thread-43 :29951 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-03-31 07:31:50,359 INFO    Thread-43 :29951 [backend.py:ensure_launched():219] starting backend process...
2022-03-31 07:31:50,415 INFO    Thread-43 :29951 [backend.py:ensure_launched():224] started backend process with pid: 4373
2022-03-31 07:31:50,417 INFO    Thread-43 :29951 [wandb_init.py:init():501] backend started and connected
2022-03-31 07:31:50,437 INFO    Thread-43 :29951 [wandb_run.py:_config_callback():992] config_cb None None {'batch_size': 26, 'criterion': 'loge', 'device': 'cuda:2', 'epochs': 25, 'hidden_dim': 512, 'learning_rate': 4.594822028254274e-05, 'num_heads': 1, 'num_hidden': 2, 'optimizer': 'adam', 'similarity_threshold': 0.6}
2022-03-31 07:31:50,443 INFO    Thread-43 :29951 [wandb_run.py:_label_probe_notebook():947] probe notebook
2022-03-31 07:31:50,443 INFO    Thread-43 :29951 [wandb_run.py:_label_probe_notebook():957] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-03-31 07:31:50,443 INFO    Thread-43 :29951 [wandb_init.py:init():565] updated telemetry
2022-03-31 07:31:50,534 INFO    Thread-43 :29951 [wandb_init.py:init():596] communicating run to backend with 30 second timeout
2022-03-31 07:31:51,631 INFO    Thread-43 :29951 [wandb_run.py:_on_init():1759] communicating current version
2022-03-31 07:31:51,709 INFO    Thread-43 :29951 [wandb_run.py:_on_init():1763] got version response 
2022-03-31 07:31:51,710 INFO    Thread-43 :29951 [wandb_init.py:init():625] starting run threads in backend
2022-03-31 07:31:53,198 INFO    Thread-43 :29951 [wandb_run.py:_console_start():1733] atexit reg
2022-03-31 07:31:53,199 INFO    Thread-43 :29951 [wandb_run.py:_redirect():1606] redirect: SettingsConsole.WRAP
2022-03-31 07:31:53,199 INFO    Thread-43 :29951 [wandb_run.py:_redirect():1643] Wrapping output streams.
2022-03-31 07:31:53,200 INFO    Thread-43 :29951 [wandb_run.py:_redirect():1667] Redirects installed.
2022-03-31 07:31:53,200 INFO    Thread-43 :29951 [wandb_init.py:init():664] run started, returning control to user process
2022-03-31 09:43:38,260 INFO    Thread-43 :29951 [wandb_run.py:finish():1539] finishing run leonardtd/cloud classification/925yo3hw
2022-03-31 09:43:38,260 INFO    Thread-43 :29951 [jupyter.py:save_history():429] not saving jupyter history
2022-03-31 09:43:38,261 INFO    Thread-43 :29951 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-03-31 09:43:38,261 INFO    Thread-43 :29951 [wandb_init.py:_jupyter_teardown():337] cleaning up jupyter logic
2022-03-31 09:43:38,261 INFO    Thread-43 :29951 [wandb_run.py:_atexit_cleanup():1702] got exitcode: 1
2022-03-31 09:43:38,263 INFO    Thread-43 :29951 [wandb_run.py:_restore():1674] restore
2022-03-31 09:43:38,515 INFO    Thread-43 :29951 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 910
  total_bytes: 910
}

2022-03-31 09:43:38,820 INFO    Thread-43 :29951 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 910
  total_bytes: 910
}

2022-03-31 09:43:39,773 INFO    Thread-43 :29951 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 910
  total_bytes: 100906
}

2022-03-31 09:43:39,877 INFO    Thread-43 :29951 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 910
  total_bytes: 100906
}

2022-03-31 09:43:39,979 INFO    Thread-43 :29951 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 910
  total_bytes: 100906
}

2022-03-31 09:43:40,082 INFO    Thread-43 :29951 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 910
  total_bytes: 100906
}

2022-03-31 09:43:40,184 INFO    Thread-43 :29951 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 910
  total_bytes: 100906
}

2022-03-31 09:43:40,287 INFO    Thread-43 :29951 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 92701
  total_bytes: 100906
}

2022-03-31 09:43:40,391 INFO    Thread-43 :29951 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 100906
  total_bytes: 100906
}

2022-03-31 09:43:40,495 INFO    Thread-43 :29951 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 100906
  total_bytes: 100906
}

2022-03-31 09:43:40,598 INFO    Thread-43 :29951 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 100906
  total_bytes: 100906
}

2022-03-31 09:43:40,700 INFO    Thread-43 :29951 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 100906
  total_bytes: 100906
}

2022-03-31 09:43:40,803 INFO    Thread-43 :29951 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 100906
  total_bytes: 100906
}

2022-03-31 09:43:40,905 INFO    Thread-43 :29951 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 100906
  total_bytes: 100906
}

2022-03-31 09:43:41,008 INFO    Thread-43 :29951 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 100906
  total_bytes: 100906
}

2022-03-31 09:43:41,110 INFO    Thread-43 :29951 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 100906
  total_bytes: 100906
}

2022-03-31 09:43:41,213 INFO    Thread-43 :29951 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 100906
  total_bytes: 100906
}

2022-03-31 09:43:41,730 INFO    Thread-43 :29951 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 100906
  total_bytes: 100906
}

2022-03-31 09:43:42,378 INFO    Thread-43 :29951 [wandb_run.py:_on_finish():1831] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 100906
  total_bytes: 100906
}
local_info {
}

2022-03-31 09:43:43,513 INFO    Thread-43 :29951 [wandb_run.py:_footer_history_summary_info():2865] rendering history
2022-03-31 09:43:43,514 INFO    Thread-43 :29951 [wandb_run.py:_footer_history_summary_info():2894] rendering summary
2022-03-31 09:43:43,520 INFO    Thread-43 :29951 [wandb_run.py:_footer_sync_info():2822] logging synced files
